app/models/vm_or_template.rb

```diff
@@ -151,6 +151,7 @@ class VmOrTemplate < ApplicationRecord
   virtual_column :v_owning_blue_folder,                 :type => :string,     :uses => :all_relationships
   virtual_column :v_owning_blue_folder_path,            :type => :string,     :uses => :all_relationships
   virtual_column :v_datastore_path,                     :type => :string,     :uses => :storage
+  virtual_column :v_folder_path_no_non_display_folders, :type => :string,     :uses => :all_relationships
```

Comment:
All of the other virtual_columns for the parent_blue_folders are called `parent_blue_folder_something` [ref](https://github.com/ManageIQ/manageiq/blob/552e78233afd04f0be03554c1d5ab44bf5d4ae4a/app/models/vm_or_template.rb#L267) so to keep it consistent how about `v_parent_blue_folder_display_path` ?

Reply:
Sure... as long as the name fits in that space that the other virtual columns are using 
-- and it does!

I have renamed the method and the virtual column per your suggestion.
Thanks.